templates/platforms/android-studio/app/src/main/kotlin/_domain_/_projectname_/RustWebChromeClient.kt.hbs

```diff
@@ -10,7 +10,7 @@ class RustWebChromeClient: WebChromeClient() {
     if (url.endsWith("##")) {
       url = url.dropLast(2)
     }
-    if (loadedUrl != url) {
+    if (loadedUrl != url && progress >= 50) {
```

Comment:
How about `progress == 100`? Looks like this function will be called multiple times.
And from my tries, it seems `100` would always be called.

Reply:
I'll check if progress == 100 isn't too late

Reply:
100 is too late, the app is loaded already and the init script needs to run earlier. changing to `>= 20` works though, but this method doesn't seem reliable and we should look for an alternative in the future.